data/src/main/java/com/example/data/database/dao/JobDao.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.data.database.dao
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import androidx.room.Update
+import com.example.data.database.entity.JobEntity
+
+@Dao
+interface JobDao {
+
+    @Query("SELECT * FROM `jobs`")
+    fun getAllJobs(): List<JobEntity>
+
+    @Query("SELECT * FROM `jobs` Where `id` in (:ids)")
+    fun getAllJobsByIdIn(ids: List<Int>): List<JobEntity>
+
+    @Query("SELECT * FROM `jobs` Where `id` = :id")
+    fun getJobById(id: Int): JobEntity?
```

Comment:
Mấy cái query này có vẻ không cần

```suggestion
    @Query
    fun getAllJobs(): List<JobEntity>

    @Query
    fun getAllJobsByIdIn(ids: List<Int>): List<JobEntity>

    @Query
    fun getJobById(id: Int): JobEntity?
```

Reply:
xoá sẽ lỗi